src/modules/roc_sndio/driver_info.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_sndio/driver_info.h
+//! @brief Driver info interface.
+
+#ifndef ROC_SNDIO_DRIVER_INFO_H_
+#define ROC_SNDIO_DRIVER_INFO_H_
+
+namespace roc {
+namespace sndio {
+
+//! Driver info interface.
+struct DriverInfo {
+    DriverInfo();
```

Comment:
It seems that `DriverInfo()` is not implemented and is never used.

Reply:
Array requires me to have a constructor declaration like this, it calls it using T().  That's the only reason why I have this, otherwise it won't compile.

Reply:
Ah, I see. Than I suggest to implement it (and zeroise `name` or `name[0]` in it).